components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -103,6 +103,15 @@ class BrowserToolbar @JvmOverloads constructor(
             }
         }
 
+    /**
+     * Sets/gets indicates if this toolbar belongs to an external session e.g Custom tab or PWA.
+     */
+    override var isExternal: Boolean = false
+        set(value) {
+            displayToolbar.shouldSeparatorBeVisible = !value
+            field = value
+        }
+
```

Comment:
I think "is external" is a too high level concept for the toolbar. Especially since it only controls the separator and not all other aspects of an external toolbar. An app setting this does not get full "external app" behavior. And whether that should show a separator or not is up to the app and at the end of the day unrelated to whether this is an external session or not.

Can this just be a flag to control the separator?

Reply:
Agree, thanks for the suggestions!
Fixed ✅